Dockerfile

```diff
@@ -1,5 +1,16 @@
 FROM quay.io/ukhomeofficedigital/nodejs-base:v8
 
+ENV USER user_hocs_frontend
+ENV USER_ID 1000
+ENV GROUP group_hocs_frontend
+
+RUN groupadd -r ${GROUP} && \
+    useradd -r -u ${USER_ID} -g ${GROUP} ${USER} -d /app && \
+    mkdir -p /app && \
+    mkdir -p /tmp && \
```

Comment:
`/tmp` is a system directory -- probably shouldn't fiddle with it unless you must.